src/components/MyProfile/MyGoal.tsx

```diff
@@ -0,0 +1,162 @@
+import { useState } from 'react';
+import styled from 'styled-components';
+import { IcAdd, IcMinusWhite } from '../../assets';
+
+const MyGoal = () => {
+  const [number, setNumber] = useState(0);
+  const [isSaved, setIsSaved] = useState(false);
+
+  const handleSaveBtnClick = () => {
+    setIsSaved(true);
+  };
+
+  const handleCancelBtnClick = () => {
+    setIsSaved(false);
+  };
+
+  const isActionAllowed = () => !isSaved;
```

Comment:
P1: 함수로 정의한 특별한 이유가 없다면, 변수로 정의하는 것이 좋을 것 같습니다
```suggestion
  const isActionAllowed = !isSaved;
```

그럼 아래도 함수 형태가 아닌 단순 변수 형태로 사용할 수 있엉ㅅ